### PR TITLE
ARSN-26 interrogate the default region for getBucketLocation

### PR DESCRIPTION
**Pull request by nicolas2bert**
Cloudserver tests pass with the change: https://github.com/scality/cloudserver/compare/bugfix/CLDSRV-39/arsenal

### CODE REVIEW

**Comment by nicolas2bert** [comment]
ping

File: lib/storage/data/external/AwsClient.js [hunk 1]
```diff
@@ -30,6 +30,9 @@ class AwsClient {
     }
 
     setup(cb) {
+        // NOTE: to get the bucket location we interrogate the default
```
Comment (naren-rajendran) [inline lib/storage/data/external/AwsClient.js:33]: what does interrogate mean here?

**Review by miniscruff — APPROVED** [review]

File: lib/storage/data/external/AwsClient.js (hunk 1 above)
Comment (nicolas2bert) [reply lib/storage/data/external/AwsClient.js:33]: obtain data from/accesses the resources in that geographical area.

**Review by naren-rajendran — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e approve

**Pull request merged**